rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/locking/MultiReadSingleWriteLock.java

```diff
@@ -55,11 +54,8 @@ public boolean acquireWriteLock(String ownerId) {
                 Thread.sleep(250);
             }
         } catch (InterruptedException e) {
-            Throwables.propagate(e);
+            throw new RuntimeException(e);
```

Comment:
I realize it's a change in behavior, but you should probably call `Thread.currentThread().interrupt()`, as above when catching `InterruptedException`

Reply:
hmm let's see what happens!